### ISSUE TITLE
[OMP] Initial manuscript submission form is intimidating

### ISSUE THREAD

**Issue by crism** [pkp/pkp-lib#1143]
From #1127, against `HEAD`:

The initial submission form is intimidating. I believe that the form is the same as the one used for catalog information; it should be split apart, so the submission form can be separately configured per press preference.

**Comment by asmecher** [comment]
@NateWr and @stranack -- we talked about having the metadata form hidden by default, with a "More Fields" type control to show all the fields. Personally I think this will just decrease the amount of metadata provided, but perhaps that's a worthwhile trade-off. @stranack, I think this was raised in your UI/UX review; did it get filed elsewhere?

**Comment by stranack** [comment]
In addition to possibly hiding them, there is also a recommendation to 
let JM's turn them off or on in the Setup -- see line 12:

Submission: Allow JM to remove any/all article metadata fields (as in OJS 2)

I'd suggest turning most of them OFF by default, except Keywords, and 
let JM's enable any they need. I think this will be an important part of 
making the submission process significantly more user-friendly.

Kevin

On 2016-02-16 12:33 PM, Alec Smecher wrote:
> @NateWr <https://github.com/NateWr> and @stranack
> <https://github.com/stranack> -- we talked about having the metadata
> form hidden by default, with a "More Fields" type control to show all
> the fields. Personally I think this will just decrease the amount of
> metadata provided, but perhaps that's a worthwhile trade-off. @stranack
> <https://github.com/stranack>, I think this was raised in your UI/UX
> review; did it get filed elsewhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkp/pkp-lib/issues/1143#issuecomment-184864616>.
>

-- 
--
Kevin Stranack, MLIS, MAEd
Learning and Development Consultant
Email: kevin@stranack.ca
Twitter: @stranack
Skype: kstranack
Web: http://stranack.ca

**Comment by asmecher** [comment]
Making the fields optional was what we did in OJS 2.4.x, and we decided to remove that functionality for 3.0 ;) Not to say we can't revisit the decision...

**Comment by stranack** [comment]
I'd really like to revisit that decision. I wasn't involved so can't 
know the reasons, but I don't think it is realistic to expect authors to 
answer all of these non-required fields. But keeping them available to 
turn ON allows the decision to rest with the JM/editor, not us.

**Comment by NateWr** [comment]
_tldr: Let's make an effort to educate on, and receive meaningful feedback about, the costs vs benefits of metadata collection during submission. Then let's commit to a best-practice default with flexible on-off toggles for customization._

I think this is a pretty important issue. From a user's perspective, the cost of all these metadata fields is very high. But I acknowledge that for editorial/administrative staff the benefits of receiving data may also be very high.

I think there are a few overlapping concerns to work out here.

# Problems for the user

**Many of the fields are bewildering**
The user tests that @stranack completed showed that users did not understand what the fields were for or what data was meant to go into them. Data that was entered was frequently wrong and the help information provided, such as that pointing to technical specifications for Dublin core, only made things worse by confusing them further.

**The number of fields is overwhelming**
Getting confronted by the most complex step of the submission process right up front was really disheartening. As one user articulated during the tests (but many expressed in other ways), they didn't feel "committed" yet. As soon as they realized the fields were optional, they let out a sigh of relief.

**The fields are prone to getting bad data**
Because the fields are confusing for many users, they're likely to collect inaccurate data. Attentive editors can edit this, of course, but I worry about lesser-resourced groups (particularly with OJS) who may usher content through the workflow without ever knowing the metadata is in there. We may also be creating more work for editors who need to "clean out" bad metadata fields they don't normally use.

**The stakes are very high for submitters**
Someone submitting a manuscript or article is not your average Facebook user updating their status. They're a professional -- maybe an early-career academic -- for whom the submission process has serious implications. That means, on the one hand, that they're more willing to struggle through a difficult process. But it also means that their anxiety levels and retisence to "try things out" in the UI are already pretty high. Presenting a bunch of very technical fields that they don't understand at this stage feels like a mean approach.

# Where editor/publisher concerns may diverge

It may be that publishers want the barrier to submission to be pretty high. Many are already overwhelmed by the volume of submissions and may want a certain level of committment to be demonstrated before they have to evaluate the submission.

Many may also want the author to provide the metadata so that they don't have to. Even if the metadata needs adjusting to fit the standards, they may want "recommendations" from the author(s).

This is speculative, though, and will likely differ from publisher to publisher. There may be other reasons here I'm not aware of (maybe you have some ideas @asmecher?).

# Suggested approach

I would recommend that we tackle this problem from multiple angles (technical and consultative) and come up with an approach that PKP would feel happy recommending as a best practice. I would suggest this includes the following:

1. Reach out to stakeholders to hear from them the costs v. benefits of metadata and how its routinely entered/handled. How important is the data? Do they get good data from submitters? When during the workflow is the metadata valuable? What's the comparitive value of submission metadata vs file metadata? In addition to the PKP community, I think it would be helpful to reach out to friendly institutional actors who have worked through this problem themselves at a sizeable scale, particularly those who run end-to-end platforms where they interact closely with administrative, editorial and technical staff (eg - PLOS, eLife, Ubiquity).

2. Work on changes to the UX flow which might reduce the burden of metadata input. The multi-upload control I'm planning for OJS 3.0 will probably push file metadata entry a little later in the submission workflow. We might also consider how we could provide some way for an editor to prompt the author to input metadata after the initial submission (eg - after they've checked the file format but before it's sent for review or after it passes review). This would allow a smoother submission process without metadata, while still providing editors a way to require this before their investment gets too large.

3. Prepare optional on/off toggles for metadata so that there is flexibility while we are pursuing the consultative process. When we have the results of that process, we can implement best practice defaults while still providing that flexibility.

# Timeline

Of course, this won't be complete in time for OMP 1.2. And I won't get arround to revising the submission workflow either #975. The obvious target is OJS 3.0.

In the meantime, I land with @stranack in terms of the default being OFF and allowing admins/editors to enable the fields they want. For OMP 1.2 this can be a global enable/disable of metadata fields, and we can leave the question of whether or not to have a distinct behaviour set for submission itself to a later date.

I do think that so far the feedback we've had has been that there are too many fields, but the user tests were primarily with administrators. Maybe editors feel differently.

Also, I'm happy to hear more about what's behind the prior decision to not make them optional.

**Comment by asmecher** [comment]
Thanks, @NateWr and @stranack. I agree that we have a problem that needs solving, but returning wholesale to the OJS 2.x approach won't solve it -- editors will either disable a bunch of fields and limit their indexing capabilities, or authors will continue to be overwhelmed and provide bad data. Some ideas:

- Put some effort into how the fields are presented. For example, hide most of them by default, and permit power users to spin out more if they want them, categorized logically. Provide links to e.g. DC resources to give clear guidance on how the fields can be used. This might more clearly indicate to authors that the fields are not required and reduce data poisoning.
- Stop using the same forms for submission as for metadata elsewhere in the system. Authors will be terrified by the metadata forms during submission, but I suspect they'll be less bothered by semi-populated forms later. Likewise, editors may want authors to get the basics right (title, abstract, keywords) and be content to fill in the rest themselves. Forcing wholesale enable/disable of the field in all places prevents this.

Both of these ideas might be good candidates for some mockup-based testing with authors.

**Comment by crism** [comment]
Splitting the forms between submission and elsewhere is key, I think. That allows publishers to configure them separately. In our case, we’d rather have our metadata librarians do it _correctly_, and not have the author’s bungled attempts at all, but we _do_ want to be able to add that information at the right time. Other publishers will want more information right from the start; configurability is the key here.

**Comment by stranack** [comment]
In the interest of time (which we're running out of), could we go with 
re-adding the ability to turn submission metadata fields on or off, and 
revisit at a later point? From an author's perspective, this has been 
identified as a significant UX hurdle, so I'd like to at least have a 
temporary response for the production version.

Kevin

**Comment by asmecher** [comment]
If we can, let's avoid add setup options that are likely to change meaning later. How about...

1. Moving submission process metadata form elements into a spin-out "See More Metadata..." div, or
2. Adding setup options to selectively hide submission process metadata fields, or
3. Both

Note that I'm suggesting this only for the author's initial submission process, not the metadata modal available while with workflow is in process. It's the submission process that authors are finding frightening in both testing rounds.

**Comment by stranack** [comment]
I'd vote for 2, but will shut up about it now :)

We did also get feedback from an editor who was unhappy with the 
overwhelming amount of metadata being presented in the workflow to her 
("I'm not a librarian"). I think it is still a valid question as to how 
absolutely necessary "Chronological or historical coverage", 
"Geo-Spatial Coverage", and "Sample Coverage" are (it would be 
fascinating to know what percentage of OJS journals actually use these 
fields), but am fine with putting this off for a 3.1 discussion.

Kevin

On 2016-02-18 1:49 PM, Alec Smecher wrote:
> If we can, let's avoid add setup options that are likely to change
> meaning later. How about...
>
>  1. Moving submission process metadata form elements into a spin-out
>     "See More Metadata..." div, or
>  2. Adding setup options to selectively hide submission process metadata
>     fields, or
>  3. Both
>
> Note that I'm suggesting this only for the author's initial submission
> process, not the metadata modal available while with workflow is in
> process. It's the submission process that authors are finding
> frightening in both testing rounds.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkp/pkp-lib/issues/1143#issuecomment-185936731>.
>

-- 
--
Kevin Stranack, MLIS, MAEd
Learning and Development Consultant
Email: kevin@stranack.ca
Twitter: @stranack
Skype: kstranack
Web: http://stranack.ca

**Comment by asmecher** [comment]
@willinsky, if I could haul you in on Kevin's last point; I agree that the coverage fields are extremely specific and probably rarely used properly. Currently we have `DC.Coverage` (sample coverage), `DC.Coverage.spatial`, and `DC.Coverage.temporal`; would we be losing anything by just using `DC.Coverage`?

**Comment by beghelli** [comment]
I like option 2 also. And thinking about it, why we don't hide everything that's not required for an initial submission process into "extra metatada" extras on demand controller? Along with that we can implement a quick listbuilder in the management so managers could list all the submission metadata required fileds. The ones that are not required will not even show up for the author at the first time, they would be hidden inside the extra metadata...

**Comment by NateWr** [comment]
I agree with @asmecher and @crism that separating the submission form from the catalog entry form will be a key part of this solution. I also think that having these things be toggle-able in the backend would be an easy task. Here's a quick mockup:

![metadata-enable](https://cloud.githubusercontent.com/assets/2306629/13215199/ac5c1fe4-d94b-11e5-9854-0db220a8d6d2.png)

I think this could/should be done for OMP 1.2. We can also dump the metadata fields into an `extrasOnDemand` controller where they appear in both forms to tidy things up a bit.

I still think we should engage in a consultative process so that we can make more meaningful decisions about the defaults for OJS 3.0 (and future OMP iterations). We can all lean towards various implementations. But we should really figure out the most common workflows and design for those, with sensible options that can cater to other needs.

**Comment by willinsky** [comment]
@asmecher The standard that has emerged for the submission requirements for journals over the decade since we introduced "Coverage" is to request of authors "Keywords" (which is Dublin Core's Subject element) and abstract or what DC calls Description. This would also holds for books (although online submission is not yet the standard for books). With books, the request for more detailed catalog information typically comes after the review and acceptance and copyediting stages, in my experience, as the books goes into production, and may involve the editor, who knows the book by this point. The editor may want to have the author sign off on the description and keywords in the Production stage, given the book may have been revised significantly in the review process.

**Comment by asmecher** [comment]
Decision for now:

- Consolidate the existing "Coverage" fields into a single field.
- Add a setup area where metadata fields can be separately enabled/disabled for the submission process, and elsewhere in the workflow.
- When that's ready, see about improving the visual presentation of the forms to reduce confusion.

**Comment by crism** [comment]
I like this plan, @asmecher. Thanks to everyone who weighed in.

**Comment by asmecher** [comment]
Pulls:
- https://github.com/pkp/omp/pull/228
- https://github.com/pkp/pkp-lib/pull/1200

Here's a screenshot of the resulting grid, which is at the bottom of Settings > Workflow > Submission:

![grid](https://cloud.githubusercontent.com/assets/200411/13366864/d4aae260-dc92-11e5-85a6-4d531b98253c.png)

**Comment by crism** [comment]
I like it.

**Comment by willinsky** [comment]
It may just be out of context and it may just be the two meanings of submission, but the submission column may make more sense to editors as “Entered on Submission” with the workflow column the “Complete Set,” with a note on how metadata will need to be added during the editorial process.

**Comment by asmecher** [comment]
Yes, we've struggled with this before ("submission" as a verb and as a noun). For now I'd suggest we cover it in Help, unless we can dig up another word for one of the two cases.

**Comment by willinsky** [comment]
I was hoping that entitling the columns “Entered on Submission” and the “Complete Set” would explain two things, both what metadata is wanted with the initial submission (could be "Entered with Submission") and what metadata is wanted in total.

**Comment by stranack** [comment]
What about "Entered by Author" or "Entered on Submission" and "Entered 
by Editor" or "Entered in Workflow"? For me at least, "Complete Set" 
isn't clear.


On 2016-02-26 5:57 PM, willinsky wrote:
> I was hoping that entitling the columns “Entered on Submission” and the
> “Complete Set” would explain two things, both what metadata is wanted
> with the initial submission (could be "Entered with Submission") and
> what metadata is wanted in total.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkp/pkp-lib/issues/1143#issuecomment-189552720>.
>

-- 
--
Kevin Stranack, MLIS, MAEd
Learning and Development Consultant
Email: kevin@stranack.ca
Twitter: @stranack
Skype: kstranack
Web: http://stranack.ca

**Comment by NateWr** [comment]
I agree the wording needs some work. My 2 cents: swap the column order and rename them to Enabled / Submission Form.

If that's not clear enough than I suggest we go with something more verbose, like: "Request during author submission"

**Comment by willinsky** [comment]
Yes, yes, yes to "swap the column order and rename them to Enabled / Submission Form."

**Comment by asmecher** [comment]
Tweaked per @NateWr and @willinsky.
